src/index.js

```diff
@@ -37,8 +39,8 @@ export default function supernova(env) {
 
       const [pageInfo, setPageInfo] = useState({ top: 0, height: 100 });
       const [muiParameters] = useState(muiSetup(__OPIONAL_THEME_DEPS__));
-
       const [tableData] = usePromise(() => manageData(model, layout, pageInfo), [layout, pageInfo]);
+      const changeSortOrder = useMemo(() => sortingFactory(model, layout), [layout]);
```

Comment:
Unsure about using useMemo here, think you could just as well call it directly.

Reply:
Also, layout could come from the TableHeader, its only model that needs to be hoisted to the function.

Reply:
Trying to avoid fetching if I don't need to, but it probably don't make any difference